### DISCUSSION TITLE
Should nuxt/bridge have LICENSE file?

### DISCUSSION THREAD

**Discussion by AndreyYolkin** [nuxt/bridge#382] [Ideas]
When nuxt/bridge was a part of monorepo, there was LICENSE file in the monorepo root. Now bridge is in the separate repository and MIT mention is only in package.json. I think, we need LICENSE file at the root of this repo too

**Comment by danielroe** [comment]
Indeed it should! PR welcome if you get to this before I do.

**Reply by AndreyYolkin** [reply]
https://github.com/nuxt/bridge/pull/383 here it go!
